packages/core/src/utils/common.ts

```diff
@@ -81,47 +84,38 @@ export class BNConverter {
   }
 }
 
+export const convertBtcTransactionDTO = (
```

Comment:
we have also `convertXXX` funtion in eth packages. why only implements bitcoin convert function into util?